vrk-kpa/suomidigi
# Suopa 316 editor in chief role

TO TEST:
- make fresh
- create a content producer user
- create an editor in chief user
- test that their permissions work as expected
